starknet_py/net/client_models.py

```diff
@@ -203,16 +203,41 @@ class InvokeTransaction(DeprecatedTransaction):
     nonce: Optional[int] = None
 
 
+@dataclass
+class InvokeTransactionV0(DeprecatedTransaction):
+    """
+    Dataclass representing invoke transaction v0.
+    """
+
+    type: str
```

Comment:
Could you remove the `type` from all classes? It's not necessary to include a field for it, as it can be deduced from the class type. Apologies for not pointing this out earlier.